scripts/dashboard-importer/README.md

```diff
@@ -0,0 +1,185 @@
+# Cloud Monitoring Dashboard Importer
+
+Convert dashboards from the Grafana dashboard JSON schema to the Cloud
+Monitoring dashboard JSON schema and upload them directly into Cloud Monitoring.
+
+## Prerequisites
+To run the dashboard importer, you must have both Node.js and gcloud installed.
+1. Node.js
+    - [Installation Docs](https://nodejs.org/en/download)
+    - Recommended min Node.js version: 20.4.1
+1. gcloud
+    - [Installation Docs](https://cloud.google.com/sdk/docs/install)
+    - [Ensure your gcloud version is up to date](https://cloud.google.com/sdk/gcloud/reference/components/update)
+    - [Obtain access credentials](https://cloud.google.com/sdk/gcloud/reference/auth/login) by running `gcloud auth login`
+
+## Set Up
+Clone the `monitoring-dashboard-samples` repo and CD into the `dashboard-importer` directory
+```
+git clone https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples.git && cd dashboard-importer
```

Comment:
cd dashboard-importer

I think this will need to be "cd scripts/dashboard-importer" in this repo.

(I thought I had this in the first set of comments, but I'm not seeing it. Sorry if it's a repeat!)

Reply:
This is a nice catch! thank you! I've fixed this!